src/themes/classic/components/index.js

```diff
@@ -4,12 +4,16 @@ import Code from './code'
 import Pane from './pane'
 import Select from './select'
 import Tooltip from './tooltip'
+import Input from './input'
+import TagInput from './tag-input'
 
 export default {
   Alert,
   Button,
   Code,
   Pane,
   Select,
-  Tooltip
+  Tooltip,
+  Input,
```

Comment:
Mind alphabetizing? (should really be automatic formatting imo)

Reply:
i'll follow up with something that'll enforce this more broadly across the eg codebase 👍